<?xml version="1.0"?>
<launch>
  <arg name="input/vector_map" default="/map/vector_map"/>
  <arg name="input/camera_info" default="/camera/camera_info"/>
  <arg name="input/route" default="/planning/mission_planning/route"/>
  <arg name="expect/rois" default="~/expect/rois"/>
  <arg name="output/rois" default="~/output/rois"/>
  <arg name="param_path" default="$(find-pkg-share autoware_traffic_light_map_based_detector)/config/traffic_light_map_based_detector.param.yaml"/>

  <node pkg="autoware_traffic_light_map_based_detector" exec="traffic_light_map_based_detector_node" name="traffic_light_map_based_detector" output="both">
    <remap from="~/input/vector_map" to="$(var input/vector_map)"/>
    <remap from="~/input/camera_info" to="$(var input/camera_info)"/>
    <remap from="~/expect/rois" to="$(var expect/rois)"/>
    <remap from="~/input/route" to="$(var input/route)"/>
    <remap from="~/output/rois" to="$(var output/rois)"/>
    <param from="$(var param_path)"/>
  </node>
</launch>
